src/EditorFeatures/Test/Collections/IntervalTreeTests.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.CodeAnalysis.Editor.UnitTests.Collections
 {
     public class IntervalTreeTests
     {
-        private class TupleIntrospector<T> : IIntervalIntrospector<Tuple<int, int, T>>
+        private readonly struct TupleIntrospector<T> : IIntervalIntrospector<Tuple<int, int, T>>
```

Comment:
aside. this coudl take a whack from teh perf stick to use things like ValueTuple instead of Tuple.  I can do that if you'd lke

Reply:
This is test code.